code/drasil-code/lib/Language/Drasil/Code/Imperative/Build/Import.hs

```diff
@@ -21,42 +21,42 @@ import Text.PrettyPrint.HughesPJ (Doc)
 -- | Holds all the needed information to run a program.
 data CodeHarness = Ch {
   buildConfig :: Maybe BuildConfig,
-  runnable :: Maybe Runnable, 
+  runnable :: Maybe Runnable,
   goolState :: GOOLState,
   progData :: ProgData,
   docConfig :: Maybe DocConfig}
 
 -- | Transforms information in 'CodeHarness' into a list of Makefile rules.
 instance RuleTransformer CodeHarness where
-  makeRule (Ch b r s m d) = [mkRule (makeS "# Project Name and Purpose") 
-    [makeS (progName m ++ " - To " ++ progPurp m)] []]  ++
-    maybe [mkRule buildTarget [] []]
+  makeRule (Ch b r s m d) = maybe [mkRule ["Project Name and Purpose: " ++ 
+    progName m ++ " - To " ++ progPurp m] buildTarget [] []]
```

Comment:
I feel like the "To" should be added to the purpose message earlier, if at all. Should we spin this into a new ticket?

Reply:
I am not sure I understand when you say the "To" should be added to the purpose message earlier. Could you please elaborate on that? We could remove " - To" as that would be in line with how the purpose is displayed in the README files. 

https://github.com/JacquesCarette/Drasil/blob/39a0a2b8f7ad6d980c1d00b4d1a6549c9d965f7f/code/stable/dblpendulum/src/csharp/README.md?plain=1#L4

Reply:
Ah, that sounds better too. I like it. @smiths ?

Reply:
Yes, that purpose sounds good to me.

Reply:
Ok, i will make that change now!